frontends/p4/simplifyDefUse.cpp

```diff
@@ -367,20 +367,34 @@ class FindUninitialized : public Inspector {
             }
         }
 
-        // Symbolically call some methods (actions and tables)
+        // Symbolically call some methods (actions and tables, extern methods)
         const IR::Node* callee = nullptr;
-        if (mcd.instance->is<ActionCall>()) {
-            auto action = mcd.instance->to<ActionCall>()->action;
-            callee = action;
+        if (auto ac = mcd.instance->to<ActionCall>()) {
+            callee = ac->action;
         } else if (mcd.instance->isApply()) {
             auto am = mcd.instance->to<ApplyMethod>();
             if (am->isTableApply()) {
                 auto table = am->object->to<IR::P4Table>();
                 callee = table;
             }
-        }
+        } else if (auto em = mcd.instance->to<ExternMethod>()) {
```

Comment:
This code seems to be the same as in def_use.cpp. Maybe it can be refactored to be written only once?